.github/workflows/test-automation.yml

```
@@ -32,15 +32,8 @@ jobs:
     - name: Run unit tests
       run: npm run test:fast
       
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)